jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -292,11 +292,11 @@ public Stream<Package> packages(ClassLoader classLoader) {
 
 	@SuppressWarnings("removal")
 	public void invalidatePackageAccessCache() {
```

Comment:
The entire method should be removed. Also for addNonExportedPackages.

Reply:
Removed this and `addNonExportedPackages` for JDK24.